oar/core/jira.py

```diff
@@ -232,6 +238,65 @@ def get_high_severity_and_can_drop_issues(self, jira_issue_keys):
 
         return high_severity_issues, can_drop_issues
 
+    def create_cvp_issue(self, abnormal_tests):
+        """
+        Create CVP issue with abnormal test details
+
+        Args:
+            abnormal_tests: details to be included in issue description
+
+        Returns:
+             JiraIssue: created issue
+        """
+        issue = self.create_issue(
+            project = "CVP",
+            summary = self.prepare_cvp_issue_summary(),
+            description = self._prepare_greenwave_cvp_jira_description(abnormal_tests),
+            issuetype = {"name": "Bug"},
+            priority = {"name": "Blocker"}
+        )
+
+        return JiraIssue(issue)
+
+    def prepare_cvp_issue_summary(self):
+        """
+        Prepare Greenwave CVP issue summary
+
+        Returns:
+             str: issue summary
+        """
+        return f"[{self._cs.release}] Greenwave CVP test failures in advisories"
```

Comment:
@LuboTerifaj we can add advisory id in issue summary, it's helpful to identify duplicate issue

Reply:
If there are failures in more advisories, the summary may be too long. It may be better to keep the details in jira description.

Reply:
if we decided to put all the failures in single ticket. then release number is good enough